.travis.yml

```diff
@@ -31,7 +31,7 @@ matrix:
         TESTS="1" ODOO_REPO="odoo/odoo"
     - python: 3.4
       env:
-        TESTS="1" ODOO_REPO="OCA/OCB"
+        TESTS="1" ODOO_REPO="OCA/OCB" VERSION="11.0"
```

Comment:
@moylop260 
What do you think about this change to avoid the fail in the build `OCA/OCB`, the branch `saas-18` is not present on [OCA/OCB](https://github.com/OCA/OCB/)

I think this change is temporal while branch 11.0 is created in this repository